qa/integration-tests/src/test/java/io/camunda/it/backup/BackupRestoreIT.java

```diff
@@ -206,6 +205,8 @@ public void shonldBackupAndRestoreToPreviousState(final BackupRestoreTestConfig
     exportingActuator.resume();
 
     // when
+    // closing down backup's thread poll before shutting down testStandaloneApplication
+    testStandaloneApplication.bean(HistoryBackupComponent.class).shutdownExecutor();
```

Comment:
:x: the executor should be closed when closing the application, no extra calls should be used